### ISSUE TITLE
Include license file

### ISSUE THREAD

**Issue by uilianries** [pfultz2/Linq#5]
Hi!

It would be nice to have a license file in your project.

**Comment by Shadowsith** [comment]
Yes please do this because it is unfortunately unclear with what restrictions this project comes and if I can use it for commecrial projects or not.

**Comment by Shadowsith** [comment]
If you see in the source code you will see that this project is licensed under the Boost Software License, Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
